docs/user-guides/creating-a-simple-ca-based-issuer.md

```diff
@@ -0,0 +1,53 @@
+# Creating a simple CA based issuer
+
+`cert-manager` can be used to obtain certificates using a CA key pair. The first two steps [here][1] from the Kubernetes documentation describe how you can generate a self-signed CA key pair using [openssl][2]. The output of these commands will be two files, `ca.key` and `ca.crt`, the key and certificate for your CA.
+
+We are going to create an `Issuer` that will use this CA to generate signed certificates. You can read more about the `Issuer` resource [here][3]. To allow the `Issuer` to reference our CA we need to put it into a `Secret`. `Issuers` are namespaced resources and can only reference and create `Secrets` in their own namespace, so we will need to make sure to put our CA `Secret` into the same namespace as our `Issuer`. We could alternatively create a `ClusterIssuer` in which case our `Secret` will need to be in the namespace specified by the `--cluster-resource-namespace` flag on the `cert-manager` controller.
```

Comment:
We could alternatively create a `ClusterIssuer` -> We could alternatively create a `ClusterIssuer`, a cluster-scoped version of an `Issuer`, ...